content/en/docs/release_notes/zigong.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Zigong release notes
+linkTitle: Zigong
+description: 5 Novermber 2021
+date: 2021-11-05
+Hide_readingtime: true
+---
+## Summary
+
+Added support for creating and editing flows in the UI with the new alpha [OAS flow-trigger](/docs/developer_guide/flows/flow_triggers/oas_flow_trigger) and added [documentation](/docs/developer_guide/flows/flow_triggers/oas_flow_trigger) with information on how to start using it and our roadmap for its development.
+
+{{% releasenotes/upgrade %}}
+
+<!-- ## Breaking changes -->
+
+## Features
+* #7104: Added the ability to create and edit flows with the alpha release of the [OAS flow-trigger](https://www.npmjs.com/package/@axway/api-builder-plugin-ft-oas). Now, flows can be created from the UI and invoked. :warning: Note that this module is currently WIP and should not be considered production ready.
+
+## Fixes
+* #7105: Response `body` is no longer logged at DEBUG to improve performance.
```

Comment:
Not enough information. Where/when does this happen. what public modules/plugins does it impact?

Reply:
Corrected it.